### PR TITLE
Test recipe page

### PR DESCRIPTION
**Pull request by nhilikeknee**
There's a bug where the recipe page still shows when we click to go home. For now, just refresh the page if you want to go home/settings. Once the router part is completed, we can fix this bug. Also the styling is not complete yet, there's still a lot of weird spacing. There's something causing a weird offset in the structure, I'll debug it later but just wanted to get a basic working version out bf discussion today.

### CODE REVIEW

File: source/scripts/RecipePage.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+      </section>
+      `;
+
+    // Header - title
+
+    this.shadowRoot.querySelector(".header > h1").innerHTML = data["title"];
+
+    //get ingredient list
+    const ingredients = getIngredients(data);
+    ingredients.forEach(ingredient => {
```
Comment (kjl001) [inline source/scripts/RecipePage.js:166]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8414136)
```suggestion
    ingredients.forEach((ingredient) => {
```

File: source/scripts/RecipePage.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+ * @param {Object} data JSON
+ * @returns {Array} return a list of ingredients
+ */
+function getIngredients(data) {
+  const steps = data["analyzedInstructions"][0]["steps"];
+  let list = [];
+  let index = 0;
+  steps.forEach((step) => {
+    let ingredients = step["ingredients"];
+    console.log('ingredients', ingredients);
```
Comment (kjl001) [inline source/scripts/RecipePage.js:201]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8414136)
```suggestion
    console.log("ingredients", ingredients);
```

**Comment by nhilikeknee** [comment]
Also as of right now, the recipes clicked isn't the recipe that is being shown. @dnthem and @tritonengineer can you take a look and make sure it functions properly for the mvp?

**Comment by dnthem** [comment]
> Also as of right now, the recipes clicked isn't the recipe that is being shown. @dnthem and @tritonengineer can you take a look and make sure it functions properly for the mvp?

when we implement the router, this should be fixed

**Pull request merged**